controllers/backstage_controller_test.go

```diff
@@ -56,10 +56,12 @@ var _ = Describe("Backstage controller", func() {
 		Expect(err).To(Not(HaveOccurred()))
 
 		backstageReconciler = &BackstageReconciler{
-			Client:      k8sClient,
-			Scheme:      k8sClient.Scheme(),
-			Namespace:   ns,
-			OwnsRuntime: true,
+			Client:         k8sClient,
+			Scheme:         k8sClient.Scheme(),
+			Namespace:      ns,
+			OwnsRuntime:    true,
+			PsqlImage:      "quay.io/fedora/postgresql-15:latest",
```

Comment:
Could you please specify how it was tested with restricted network environment?
As I understand understand (according to [previous discussion](https://github.com/janus-idp/operator/pull/83)) we specifically interested in testing SCV based installation.
Can you provide this information please.

Reply:
airgap testing:
* start a cluster
* mirror images in the CSV to that cluster according to https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.10/html-single/administration_guide/index#installing-devspaces-in-a-restricted-environment or https://docs.openshift.com/container-platform/4.13/installing/disconnected_install/installing-mirroring-installation-images.html
* "turn off the internet" via whatever proxy or DNS config works for you
* perform operator install 
* verify pods all start (including postgres and dev hub)
* verify RHDH loads (route works)